kghDataManagement/kghCatalogTemplate.csv

```diff
@@ -0,0 +1,4 @@
+﻿KGH ID,Any Field,Any Field,Old material no.,,,Any Field,Any Field,Any Field,       MA price,Any Field

```

Comment:
The custom template should just have KGH ID, old material number, and price. The header names should be more reasonable and you should just match on header name. You may also want to note the header is mandatory